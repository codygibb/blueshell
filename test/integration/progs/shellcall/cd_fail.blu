cd ".tmp/no_exist" {
	print "this should not be printed"
}
print "we should have crashed already"
