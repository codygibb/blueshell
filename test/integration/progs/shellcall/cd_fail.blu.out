error: Dir_not_found .tmp/no_exist
